client/transactions/list/index.js

```diff
@@ -274,7 +274,8 @@ export const TransactionsList = ( props ) => {
 			},
 			// eslint-disable-next-line camelcase
 			risk_level: {
-				value: txn.risk_level,
+				value: calculateRiskMapping( txn.risk_level ),
+				actualValue: txn.risk_level,
```

Comment:
I think this is not necessary because it's not used anywhere (right?). If it is needed in the future, then we can add it later.
What do you think?

Reply:
I agree, we can remove it, for now, we can add it later if needed.